src/main/java/se/sundsvall/snailmail/integration/db/model/Recipient.java

```diff
@@ -0,0 +1,45 @@
+package se.sundsvall.snailmail.integration.db.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder(setterPrefix = "with")
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "request", uniqueConstraints = {
+	@UniqueConstraint(name = "uq_request_recipient", columnNames = "recipient_id")
+})
+public class Recipient {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	Long id;
+
+	String givenName;
+
+	String lastName;
+
+	String co;
+
+	String adress;
+
+	String postalCode;
+
+	String city;
```

Comment:
Might want to make these private? 

Also use @Column annotation to set the names of the columns in the database, this also applies to the other new entities.